include/deal.II/matrix_free/dof_info.templates.h

```diff
@@ -262,14 +263,25 @@ namespace internal
         if (faces[n_exterior_faces].cells_exterior[0] ==
             numbers::invalid_unsigned_int)
           break;
+
+      unsigned int n_boundary_faces = 0;
```

Comment:
Could you please add a short comment? Also, I think the name `n_exterior_faces` that we used before is misleading. Could `n_inner_faces` be a better name?

Reply:
Yes, that is a better name. I have adjusted the name and added a comment!